src/rules/value-no-redundant-longhand/README.md

```diff
@@ -0,0 +1,58 @@
+# value-no-redundant-longhand
+
+Disallow redundant values in longhand properties.
```

Comment:
Oops, I just saw your suggestions @davidtheclark. Doesn't the "longhand" bit refer to the values rather than the properties? 

I think with the updated lead example, this is a bit clearer.